package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ftlabs-email-signatures",
-  "version": "0.0.1",
+  "version": "1.2.0",
```

Comment:
We went past 1.0.0 a little while ago (live version is at 1.1.0) for some reason this hasn't made it's way into the git repos